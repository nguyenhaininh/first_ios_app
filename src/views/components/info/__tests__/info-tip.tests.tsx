/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import InfoTip from '../info-tip';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<InfoTip/>).toJSON();
  expect(tree).toMatchSnapshot();
});
