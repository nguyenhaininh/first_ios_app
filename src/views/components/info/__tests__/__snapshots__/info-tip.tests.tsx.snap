// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "height": 25,
        "justifyContent": "center",
        "width": 70,
      },
      Object {
        "backgroundColor": "#004098",
      },
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 11,
        "fontWeight": "bold",
      }
    }
  >
    [missing "en.information" translation]
  </Text>
</View>
`;
